plugins/woocommerce-admin/client/marketplace/contexts/subscriptions-context.tsx

```diff
@@ -55,6 +55,21 @@ export function SubscriptionsContextProvider( props: {
 	};
 
 	useEffect( () => {
+		/**
+		 * Check if we have &install=PRODUCT_KEY in the URL. This means we have just
+		 * installed a new product and nwe need to refresh the list.
+		 */
+		const urlParams = new URLSearchParams( window.location.search );
+		const installKey = urlParams.get( 'install' );
+
+		if ( installKey ) {
+			refreshSubscriptions().finally( () => {
+				setIsLoading( false );
+			} );
```

Comment:
Not sure if I need to `catch` this, to show a notice in case of a failure. I think user won't even know we're doing this, so it might be confusing.

Reply:
Makes sense, we can use `addNotice` to show an error. Let's just do it for both, `loadSubscriptions` and `refreshSubscriptions`.

Reply:
OK, catch them both and I show a notice. I didn't put any action to notice like **Try again**. Needs a refactor and I didn't want to go there.